README.md

```diff
@@ -14,7 +14,7 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
 
 * Azure DevOps -> GitHub Enterprise Cloud migrations
 * GitHub Enterprise Cloud -> GitHub Enterprise Cloud migrations
-* GitHub Enterprise Server (version 3.4 or newer) -> GitHub Enterprise Cloud migrations
+* GitHub Enterprise Server (version 3.4.1 or newer) -> GitHub Enterprise Cloud migrations
```

Comment:
I'm not positive on the patch. I think @juanmrad has more of a pulse on the enterprise release

Reply:
yes. 3.4.1 is target patch.